spec/features/diff_viewer_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require "#{Rails.root}/lib/replica"
+
+describe 'diff viewer', type: :feature, js: true do
+  let(:en_wiki) { Wiki.get_or_create(language: 'en', project: 'wikipedia') }
+  let(:course) { create(:course, id: 1, start: '2013-01-01', end: '2014-01-02') }
```

Comment:
No need for the course ID here. In most cases, you shouldn't need any hard-coded record IDs at all in a spec.

Reply:
What's with these course dates? They don't seems to have anything to do with the revisions you're looking for in the spec.

Reply:
Removed the unnecessary stuff